lib/managers/files.js

```diff
@@ -1344,6 +1375,53 @@ Files.prototype.getRepresentationInfo = function(fileID, representationType, cal
 		})
 		.asCallback(callback);
 };
+
+/**
+ * Get the contents of a representation of a file, e.g, the binary content of an image or pdf.
+ *
+ * API Endpoint: '/files/:fileID?fields=representations'
+ * Method : GET
+ *
+ * @param {string} fileID The file ID to get the representation of
+ * @param {string} representationType The X-Rep-Hints type to request
+ * @param {Object} [options] Optional parameters
+ * @param {string} [options.assetPath] Asset path for representations with multiple files
+ * @param {Function} [callback] Passed a stream over the representation contents if successful
+ * @returns {Promise<Readable>} A promise resolving to a stream over the representation contents
+ */
+Files.prototype.getRepresentationContent = function(fileID, representationType, options, callback) {
+
+	options = Object.assign({ assetPath: '' }, options);
+
+	return this.getRepresentationInfo(fileID, representationType)
+		.then(reps => {
+
+			var repInfo = reps.entries.pop();
```

Comment:
I'm assuming you're expecting 0 or 1 entries because you expect `X-Rep-Hints` to only ask for a single thing? That should be mentioned in the function docs above, or better, it should be enforced somehow.

Note: my javascript isn't ideal, but I'm assuming `pop` removes the last element if there is one and returns javascript's null value if there is not. If that's the case, since the ordering of the returned list isn't specified, this is only deterministic for 0 or 1 entries.

Reply:
I think the non-determinism is acceptable — we can document around it.  I'm not especially keen on trying to do validation on their string input in the SDK.

Reply:
Documentation is sufficient to assuage me. When you update the docs, just add me to the review/change, however that works.